### PR TITLE
CNV-5920 - updating 2.5 version numbers

### PR DESCRIPTION
**Pull request by aburdenthehand**
Updating variable and hard-coded version numbers for 2.5 release.

[CNV-5920](https://issues.redhat.com/browse/CNV-5920)

### CODE REVIEW

File: modules/virt-document-attributes.adoc [hunk 1]
```diff
@@ -13,9 +13,9 @@
 :ProductRelease:
 :ProductVersion:
 :VirtVersion: 2.5
-:KubeVirtVersion: v0.30.5
+:KubeVirtVersion: v0.34.0
```
Comment (stu-gott) [inline modules/virt-document-attributes.adoc:16]: v.0.34.1

File: modules/virt-document-attributes.adoc [hunk 2]
```diff
@@ -13,9 +13,9 @@
...
 :HCOVersion: 2.5.0
-:LastHCOVersion: 2.4.2
+:LastHCOVersion: 2.4.3
```
Comment (stu-gott) [inline modules/virt-document-attributes.adoc:18]: @tiraboschi can you confirm?

Comment (tiraboschi) [reply modules/virt-document-attributes.adoc:18]: yes, right

**Pull request merged**